packages/create-waku/src/index.ts

```diff
@@ -52,21 +76,68 @@ async function notifyUpdate() {
   }
 }
 
-async function init() {
-  let targetDir = '';
-  const defaultProjectName = 'waku-project';
+async function installTemplate({
+  root,
+  packageName,
+}: {
+  root: string;
+  packageName: string;
+}) {
+  const pkg = {
+    name: packageName,
+    version: '0.0.0',
+  };
+
   const templateRoot = path.join(
     fileURLToPath(import.meta.url),
     '../../template',
   );
+  // is it nesessary?
```

Comment:
Not really necessary in production. But will error if someone use macOS and debug locally